azext_iot/operations/hub.py

```diff
@@ -1392,7 +1392,7 @@ def _iot_hub_monitor_events(cmd, interface=None, pnp_context=None,
                      output=output,
                      content_type=content_type,
                      devices=device_ids,
-                     interface_id=interface,
+                     interface_name=interface,
```

Comment:
we should probably change `...operations.hub._iot_hub_monitor_events`  parameter `interface` to `interface_name` as well.

Reply:
Changed